src/pages/gateway/headers.md

```diff
@@ -124,6 +124,65 @@ Mesh owners can use the `responseConfig.headers` object to add response headers.
     }
 ```
 
+#### Forward headers
+
+The `responseConfig.headers` object also allows you to return header values from a source. In the following example, we are requesting the `X-Magento-Cache-Id` and `X-Cache` headers from the Venia source.
```

Comment:
```suggestion
The `responseConfig.headers` object also allows you to return header values from a source. The following example requests the `X-Magento-Cache-Id` and `X-Cache` headers from the Venia source.
```

I'm making this suggestion because it's the only instance in this topic that you use "we". In general, I don't have a problem with this construction, but consistency is good.